source/fundamentals/crud/run-command.txt

```diff
@@ -0,0 +1,167 @@
+.. _golang-run-command:
+
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a **database command** with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+Compose a Command
+-----------------
+
+The {+driver-short+} provides the following methods to run database
+commands: 
+
+- ``RunCommand()``, which returns the command response as a
+  ``SingleResult`` type. You can use this method with any database command.
+- ``RunCommandCursor()``, which returns the command response as a
+  ``Cursor`` type. You can use this method if your database command
+  returns multiple result documents.
+
+Each method requires a Context and a command document. The
+command document must be an order-preserving type such as ``bson.D``.
+The following code shows how you can use the ``RunCommandCursor()``
+method to run the ``listCollections`` command on a database:
+
+.. code-block:: go
+
+   command := bson.D{{"listCollections", 1}}   
+   cursor, err := db.RunCommandCursor(context.TODO(), command)
+
+For a full list of database commands, see the :ref:`Additional
+Information <addl-info-runcommand>`.
+
+.. note:: Read Preference
+
+   ``RunCommand()`` and ``RunCommandCursor()`` do not obey the read
+   preference set for the database. You can set a read preference for a
+   database command by passing a ``RunCmdOptions`` type to either method:
+   
+   .. code-block:: go
+
+      opts := options.RunCmd().SetReadPreference(readpref.Primary())
+      err := db.RunCommand(context.TODO(), command, opts).Decode(&result)
+   
+   For more information on
+   read preference options, see the :ref:`golang-write-read-pref`
+   fundamentals page.
+
+Response
+--------
+
+Each method returns a document or a cursor containing documents with the
```

Comment:
In many cases, I like the idea of making it convenient to reference everything on a single page. From doing some research, it looks like these fields are the same as https://github.com/10gen/docs-mongodb-internal/blob/master/source/reference/method/db.runCommand.txt

I haven't checked how the Go driver handles RunCommand, but if it is just returning what the manual page references, then it might be best to direct the reader to the server documentation relevant to the version of the server they're connecting to. That would make maintenance easier for us in case it differs by version.